cscs-checks/libraries/boost/boost_python_check.py

```diff
@@ -1,20 +1,18 @@
 import os
 
+import reframe as rfm
 import reframe.utility.sanity as sn
-from reframe.core.pipeline import RegressionTest
 
 
-class BoostCrayGnuPythonTest(RegressionTest):
-
-    def __init__(self, boost_version, cray_gnu_version, python_version,
-                 **kwargs):
-
+@rfm.parameterized_test(['1.65.0', '17.08', '2.7'], ['1.65.0', '17.08', '3.6'])
+class BoostCrayGnuPythonTest(rfm.RegressionTest):
+    def __init__(self, boost_version, cray_gnu_version, python_version):
         def normalize(s):
             return s.replace('.', '_')
 
         super().__init__('boost_%s_cray_gnu_%s_python_%s_check' % (
             normalize(boost_version), normalize(cray_gnu_version),
-            normalize(python_version)), os.path.dirname(__file__), **kwargs)
+            normalize(python_version)))
```

Comment:
Set the name explicitly, not by passing it to the base class constructor. It may be deprecated in the future.

Reply:
And btw I don't think that you need to set it at all. The default name should be fine.